content/html/concepts/tags/terms/kbd/kbd.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: '<kbd>'
+Description: 'Makes charecters look like keys on a keyboard'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<kbd>` tag makes characters look like they are keys on a keyboard.
+
+## Syntax
+
+All that is needed to be done is put the characters between an opening and closing tag. The `<kbd>` tag has no attributes. They can be nested but the result will be the same.
+
+```pseudo
+<kbd><!-- Text goes here --></kbd>
+```
+  
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.</p>
+```
+
+Result:
+
+![picture of the <kbd> in action](https://raw.githubusercontent.com/ethan-master-coding/docs/kdb-tag/media/kdb%20tag%20example%201.png)
```

Comment:
The file names for the images need to be changed to something like "kbd-tag-image.png" where  "-" is used for spacing. The image link will look something like this "https://raw.githubusercontent.com/Codecademy/docs/main/media/kbd-tag-image.png".  Take a look in the media folder to see how the names are written. Or you can take a look at the [recently merged  HTML `<span>` entry](https://github.com/Codecademy/docs/pull/605/files) which uses an image.